core/modules/wbase/Task.h

```diff
@@ -123,12 +125,20 @@ class Task : public util::Command {
     // Note that manpage spec of "26 bytes"  is insufficient
 
     void cancel();
-    bool getCancelled(){ return _cancelled; }
+    bool getCancelled() { return _cancelled; }
```

Comment:
make it const?

I seriously dislike "get*" method names, but this is probably just me.